lib/dry/logic/predicates.rb

```diff
@@ -206,6 +208,7 @@ def is?(left, right)
         def not_eql?(left, right)
           !left.eql?(right)
         end
+        deprecate :eql?, :not_eql?
```

Comment:
Shouldn't this be underneath the `is_eql?` method and read `deprecate :sql?, :is_eql?`